--- src/foecommand.h.old    2009-11-22 16:45:53.067851262 -0500
+++ src/foecommand.h    2009-11-22 16:46:08.334873177 -0500
@@ -36,10 +36,10 @@
   
   virtual void doChangeDirection(double d);
   virtual void doChangeSpeed(double s);
-  virtual void FoeCommand::doAccelX(double ax);
-  virtual void FoeCommand::doAccelY(double ay);
-  virtual double FoeCommand::getBulletSpeedX();
-  virtual double FoeCommand::getBulletSpeedY();
+  virtual void doAccelX(double ax);
+  virtual void doAccelY(double ay);
+  virtual double getBulletSpeedX();
+  virtual double getBulletSpeedY();
   
  private:
   struct foe *foe;
